test/framework/src/main/java/org/elasticsearch/bootstrap/BootstrapForTesting.java

```diff
@@ -222,6 +222,7 @@ static Map<String, URL> getCodebases() {
         addClassCodebase(codebases, "elasticsearch-cli", "org.elasticsearch.cli.Command");
         addClassCodebase(codebases, "elasticsearch-preallocate", "org.elasticsearch.preallocate.Preallocate");
         addClassCodebase(codebases, "elasticsearch-vec", "org.elasticsearch.vec.VectorScorer");
+        addClassCodebase(codebases, "elasticsearch-logstash-bridge", "org.elasticsearch.logstashbridge.LogstashBridge");
```

Comment:
This shouldn't be necessary? It only applies to tests, and when running with security manager, but the bridge shouldn't need ESTestCase or to run with security manager right?

Reply:
```suggestion
```